app/controllers/calagator/application_controller.rb

```diff
@@ -28,6 +29,10 @@ def recaptcha_enabled?
 
     protected
 
+    def spam_detector_enabled?
+      !ENV["GPTURK_SPAM_MODEL_ID"].to_s.empty?
```

Comment:
This could be shortened to just `ENV["GPTURK_SPAM_MODEL_ID"].present?`. ActiveSupport will handle both `nil` and `""` correctly.